specification/maps/resource-manager/Microsoft.Maps/preview/2021-12-01-preview/maps-management.json

```diff
@@ -944,8 +1039,73 @@
         }
       }
     },
+    "AccountSasParameters": {
+      "description": "Parameters used to create an account Shared Access Signature (SAS) token. The REST API access control is provided by Azure Maps Role Based Access (RBAC) identity and access.",
+      "type": "object",
+      "properties": {
+        "signingKey": {
+          "type": "string",
+          "description": "The Map account key to use for signing.",
+          "enum": [
+            "primaryKey",
+            "secondaryKey"
+          ],
+          "x-ms-enum": {
+            "name": "SigningKey",
+            "modelAsString": true
+          }
+        },
+        "principalId": {
```

Comment:
maybe a stupid question, does it need `tenantId`?

Reply:
Good question, no it doesn't. Since the principal is homed to a single tenant as a user assigned identity to the resource. There is no need to apply any tenant id as supplemental information. We validate with a 400 Bad Request if the principal id does not exist on `MapAccount.Identity.UserAssignedIdentities`.